src/zkevm_specs/evm/typing.py

```diff
@@ -615,3 +648,141 @@ def _append(
         )
 
         return self
+
+
+class CopyCircuit:
+    rows: List[CopyCircuitRow]
+    pad_rows: List[CopyCircuitRow]
+
+    def __init__(self) -> None:
+        self.rows = []
+        self.pad_rows = [CopyCircuitRow(*[FQ(0)] * 18)] * 2
```

Comment:
This is an example of how padding rows work right?  Not padding in the sense that a 0 is read after the src end; but in the sense that the circuit has a fixed number of rows and maybe not all of them are used.

If that's the case, could you add a line in the markdown doc saying that padding rows are set to all 0s, and that these values always pass the constraints?